### PR TITLE
Issue40 fix crush

### PR DESCRIPTION
**Pull request by biggieman**
#40 I think only one place need fix.

### CODE REVIEW

**Comment by Jexordexan** [comment]
Thanks for fixing this! I noticed there are other places that may throw an error on a bad choice selection. Namely `this.label()` and `this.encode()` 
I'm aware that the developer can override these methods and put in their own null checks, but I believe they should work by default. I've fixed mine by just putting `choice &&` before the return value. A simple short circuit does the trick!

**EDIT**: Disregard this comment, I've tested your changes and they seem to have fixed the problems I had mentioned as well. Thanks Again @biggieman

File: example/example.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-angular.module('example', ['ui.mention']).run(function ($rootScope) {
+angular.module('example', ['ui.mention']).run(["$rootScope", function ($rootScope) {
   $rootScope.post = {
     message: 'hi there @k'
   };
-}).directive('mentionExample', function () {
+}]).directive('mentionExample', function () {
```
Comment (ProLoser) [inline example/example.js]: Is there a reason for this change?

**Review by ProLoser — CHANGES_REQUESTED** [review]
I don't think the modification to the example file is necessary. If you remove it I'll merge in this change.

**Comment by ProLoser** [comment]
Along with #35 I prefer NOT to use the more explicit injection syntax because that should be handled by a preprocessor so that the source code can be minimal. Additionally, the minified version of the example does NOT rename the variable, voiding the need for this injection syntax. Finally, there's no reason for anyone to re-process the example file itself since it's only an example.

File: example/example.js (hunk 1 above)
Comment (biggieman) [reply example/example.js]: Yes, it's not necessary for example, but on original gulpfile.js task `scripts:example` pipes ngAnnotate (like main `scripts`)

**Comment by biggieman** [comment]
@ProLoser: remove annotates from this pull and #35
@Jexordexan this functions used at ngModel rendering and we breaks render in `this.select()`.

**Pull request merged**